spec/spec_helper.rb

```diff
@@ -8,6 +8,8 @@ def horizontal_rule(width = 5)
 
 require 'puppetlabs_spec_helper/module_spec_helper'
 
+FUTURE_PARSER = ENV['FUTURE_PARSER'] == 'true' || ENV['FUTURE_PARSER'] == 'yes'
+
```

Comment:
We shouldn't need this. No need to test with Future parser at all now that we have 4.x versions.